src/components/users/userProfile/UserProfile.js

```diff
@@ -15,26 +16,24 @@ const UserProfile = () => {
   const { id } = useParams();
 
   useEffect(() => {
-      handleFindProfile()
-  })
-
-  const handleFindProfile = () => {
-    client.get(`/user/${id}`)
-    .then(res => {
-        setProfile(res.data.data.user)
-    })
-    .catch(err => console.log(err.response))
-  };
+      client
+        .get(`/user/${id}`)
+        .then((res) => {
+          setProfile(res.data.data.user);
+        })
+        .catch((err) => console.log(err.response));
+  }, []);
 
   return (
     <div>
+      <Header/>
```

Comment:
Not for now, but if every route needs to include this header component - there is probably a better way we could structure the overall app. Something to think about next week for the teams. 

Also - see @marianphillips comment - do you need to pass in a prop here?